internal/apiserver/visualization/pdf.go

```diff
@@ -2,36 +2,92 @@ package visualization
 
 import (
 	"fmt"
-	"strconv"
+	"strings"
 
 	"github.com/jung-kurt/gofpdf"
-
-	"github.com/slovak-egov/einvoice/internal/entity"
+	"github.com/lestrrat-go/libxml2"
+	"github.com/lestrrat-go/libxml2/clib"
+	"github.com/lestrrat-go/libxml2/types"
 )
 
-func Generate(inv *entity.Invoice) *File {
-	pdf := gofpdf.New("P", "mm", "A4", "")
+var lineHeight float64 = 5
+var tabSize = 4
+var font = "Arial"
+var pageHeight float64
 
-	pdf.AddPage()
+func generateLines(n types.Node, level int, pdf *gofpdf.Fpdf) error {
```

Comment:
it would be helpful if you add some comments in this function for better readability
maybe rename `n` -> `currentNode`